dfastmi/io/RiverParameterData.py

```diff
@@ -0,0 +1,674 @@
+    
+    def config_get_bool(
+        self,
+        group: str,
+        key: str,
+        default: Optional[bool] = None,
+    ) -> bool:
+        """
+        Get a boolean from a selected group and keyword in the analysis settings.
+
+        Arguments
+        ---------
+        group : str
+            Name of the group from which to read.
+        key : str
+            Name of the keyword from which to read.
+        default : Optional[bool]
+            Optional default value.
+
+        Raises
+        ------
+        Exception
+            If the keyword isn't specified and no default value is given.
+
+        Returns
+        -------
+        val : bool
+            Boolean value.
+        """
+        try:
+            str = self._config[group][key].lower()
+            val = (
+                (str == "yes")
+                or (str == "y")
+                or (str == "true")
+                or (str == "t")
+                or (str == "1")
+            )
+        except:
+            if not default is None:
+                val = default
+            else:
+                raise Exception(
+                    'No boolean value specified for required keyword "{}" in block "{}".'.format(
+                        key, group
+                    )
+                )
+        return val
+
+    def config_get_int(
+        self,
+        group: str,
+        key: str,
+        default: Optional[int] = None,
+        positive: bool = False,
+    ) -> int:
+        """
+        Get an integer from a selected group and keyword in the analysis settings.
+
+        Arguments
+        ---------
+        group : str
+            Name of the group from which to read.
+        key : str
+            Name of the keyword from which to read.
+        default : Optional[int]
+            Optional default value.
+        positive : bool
+            Flag specifying whether all integers are accepted (if False), or only strictly positive integers (if True).
+
+        Raises
+        ------
+        Exception
+            If the keyword isn't specified and no default value is given.
+            If a negative or zero value is specified when a positive value is required.
+
+        Returns
+        -------
+        val : int
+            Integer value.
+        """
+        try:
+            val = int(self._config[group][key])
+        except:
+            if not default is None:
+                val = default
+            else:
+                raise Exception(
+                    'No integer value specified for required keyword "{}" in block "{}".'.format(
+                        key, group
+                    )
+                )
+        if positive:
+            if val <= 0:
+                raise Exception(
+                    'Value for "{}" in block "{}" must be positive, not {}.'.format(
+                        key, group, val
+                    )
+                )
+        return val
+
+    def config_get_float(
```

Comment:
is this the correct method name? config makes it sound like it belongs in the Config class.